Ideas/easier-embedded-rust.md

```diff
@@ -0,0 +1,57 @@
+# An Easier Embedded Rust
+
+## Problem Statement
+
+**How could Rust make it easier for people just getting started with Rust and/or Embedded?**
+
+Rust has made embedded development better (for me) in a number of ways:
+
+* Leveraging what Rust already gives you:
+    * Tools like Cargo for build + package management
+    * Ability to run `no_std` programs easily
+    * Modern language features (modules, iterators, type system)
+    * Making it "harder to get things wrong" - use the compiler to detect errors statically
+* Building good tools:
+    * Way better debugger tools via probe-rs, and tools built on top like probe-run and probe-rs-cli
+    * Tooling for logging, like defmt
+* Reasonably good docs, and teaching materials
+
+However, Rust does not have as good of a "getting started (new to embedded and/or new to rust" story, or a story for "just want things to work (artists/hobbyists)".
```

Comment:
The first set of parentheses need to be closed. 
I feel like this sentence could be rephrased to make it flow better (?)
Despite these benefits, Rust does not have as good of a "getting started" story, neither for those new to embedded and/or new to Rust nor for artists/hobbyists who "just want things to work."
- but also "not have as good of" in comparison to some other programming language (plz specify as example)? or just in general, like "Rust doesn't have a good 'getting started' story"